web/src/components/Carousel.jsx

```diff
@@ -0,0 +1,65 @@
+import { h, Component } from 'preact';
+import Flickity from 'flickity';
+import 'flickity/css/flickity.css';
+
+export default class Carousel extends Component {
```

Comment:
Please use functional components. All other components in the application are that way and it would be good to keep standard